corehq/motech/dhis2/repeaters.py

```diff
@@ -249,7 +249,8 @@ def get_api_version(dhis2_version):
         example, the API version for DHIS 2.25 is 25.
 
 
-    .. _DHIS 2 Developer guide: https://docs.dhis2.org/master/en/developer/html/webapi_browsing_the_web_api.html#webapi_api_versions
+    .. _DHIS 2 Developer guide:
+    .. https://docs.dhis2.org/master/en/developer/html/webapi_browsing_the_web_api.html#webapi_api_versions
```

Comment:
This change breaks the [external link](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links) on line 245. (Docstrings use reStructuredText so that they can be parsed by Sphinx.)

If the linter is complaining about the line length, you can close the docstring with `"""  # noqa: E501`.

Reply:
Ah, interesting I didn't know that. Good catch! I've put the docstring back to how it was ([e874e14](https://github.com/dimagi/commcare-hq/pull/34250/commits/e874e14c549fc0c94859fa0f11949f490d93b2e6)).